### DISCUSSION TITLE
选择图片时无法拉起相册

### DISCUSSION THREAD

**Discussion by liqycs** [WatsonYao/nian_app#119] [Q&A] [answered]
在选择图片时弹出的这个，只能显示“相机截图下载”三个相册，其余相册均无法显示。
需要手动右上角三个点，打开之后再手动选择拉起系统相册APP才行，就找图片有一点点麻烦。
红米K50澎湃OS系统。
非常荣幸能用上这么棒的nian！

![Screenshot_2024-11-30-13-59-13-044_com.android.providers.media.module.jpg](https://github.com/user-attachments/assets/e0109633-7095-4cf4-8f61-6831690c3997)

**Comment by WatsonYao** [comment, marked as answer]
这个在应用层面无法优化，调的就是系统的图片选择器，你可以看看选择器或者相册相关设置，是否对外可见之类的。
